### ISSUE TITLE
Remove page template

### ISSUE THREAD

**Issue by rugk** [PrivateBin/PrivateBin#265]
I can not see/find a single up-to-date instance of PrivateBin, which still uses the visually outdated page template.

It makes no sense to keep this legacy stuff anymore and it just wastes development resources. If we want to offer another template then we'd make a new one, but that old one is useless, IMHO.

**Comment by elrido** [comment]
I think it is very valuable to keep it, as long as there are no other templates committed instead of the bootstrap one. Having multiple templates forces us to code the JS code in such a way that it is not fully dependent on a specific themes properties (like having bootstrap CSS or bootstrap modals available). This guarantees that our template system remains flexible to accommodate very different template approaches.

So, @all, please contribute some nice alternative templates, so we may consider removing the 'page' one.

**Comment by rugk** [comment]
Indeed, maybe we should create a new alternative template instead. Or any contributor…

**Comment by ronv** [comment]
Considering to port to new bootstrap version 🤔

**Comment by rugk** [comment]
Well… the new template is already bootstrap, so that would not make much sense. We would rather use a different new template without bootstrap.

**Comment by elrido** [comment]
Note to self - also lets us remove the `allow-popups` from the sanbox CSP-directive.
